docs/user/tutorials/evnt-10-nats-server-system-events.md

```diff
@@ -0,0 +1,26 @@
+# Acquiring NATS server system account credentials
```

Comment:
These are the only instructions that used to be in the "Operations" folder. Not sure if we can just add them to the tutorials or if they need their own folder. Is the target group the same? Also, the name makes me think this might rather belong to the nats repo?

Reply:
yes, this should go into nats-manager repo.

Reply:
Alright! done with https://github.com/kyma-project/nats-manager/pull/209 and https://github.com/kyma-project/eventing-manager/pull/198/commits/26ad82972ae2093e79faa0eb99220ac5b85a04f3